docs/Installation.md

```diff
@@ -36,9 +40,9 @@ export HOMEBREW_NO_INSTALL_FROM_API=1
 /bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install.sh)"
 ```
 
-This will make Homebrew install formulae and casks from the `homebrew/core` and `homebrew/cask` taps using local checkouts of these repositories instead of Homebrew’s API.
+This will make Homebrew install formulae and casks from the `homebrew/core` and `homebrew/cask` taps using local checkouts of these repositories instead of Homebrew’s API. It is rare you need to enable this setting and it can easily be enabled later after installation should it be necessary.
```

Comment:
Maybe add something here about how these are usually only necessary if you plan to contribute to one of the core repos.

Reply:
Better to encourage scoped usage after installation rather than encouraging globally setting it prior to installation IMO.

Reply:
Agreed.

Reply:
Makes sense